packages/core/src/scatter-chart.ts

```diff
@@ -7,10 +7,16 @@ import { ChartConfig, ScatterChartOptions, ChartType } from "./configuration";
 import { Tools } from "./tools";
 
 export class ScatterChart extends BaseAxisChart {
-	options: ScatterChartOptions = Tools.merge({}, Configuration.options.SCATTER);
+	options: ScatterChartOptions;
 
 	constructor(holder: Element, configs: ChartConfig<ScatterChartOptions>) {
 		super(holder, configs);
+		// initialize options
+		if (configs.options) {
+			this.options = Tools.merge({}, Configuration.options.SCATTER, configs.options);
+		} else {
+			this.options = Tools.merge({}, Configuration.options.SCATTER);
+		}
```

Comment:
This seems not necessary.

Doing:
```ts
options: ScatterChartOptions = Tools.merge({}, Configuration.options.SCATTER);
```

and then doing:
```ts
this.options = Tools.merge({}, Configuration.options.SCATTER, configs.options)
```

would be better

Reply:
The code follows how other charts initialize options, so i would have to change those as well. I'll open another PR for that.

Reply:
I see. Ok we can merge this and fix these in a different PR.

If we use the `getProperty` function in `tools.ts`:
```ts
this.options = Tools.merge({}, Configuration.options.SCATTER, getProperty(configs, "options"));
```

It would just return `null` for `configs.options` if it doesn't exist, and we'd get what we want.

Reply:
Okay, noted. Good catch btw.